src/content/docs/agents/java-agent/configuration/configuring-your-ssl-certificates.mdx

```diff
@@ -13,12 +13,29 @@ To communicate with the New Relic collector over HTTPS, you need to have the pro
 
 For SSL connection issues, please visit [SSL or connection errors (Java)](/docs/agents/java-agent/troubleshooting/ssl-or-connection-errors-java/)
 
-## Java Agent versions 6.2.0+
+## Notice of expired SSL certificates and certificate removal
+<Callout variant="warning">
+  
+  The SSL certificates that were bundled with Java agent versions 6.1.0 - 6.4.2 have expired and are no longer valid. This means that Java agent versions 6.1.0 - 6.4.2 can no longer use the certificates bundled with the agent to establish a connection with New Relic and that you must provide your own SSL certificates.
+  
+  Please see [this community forum post](https://discuss.newrelic.com/t/important-upcoming-new-relic-server-certificate-update-will-impact-most-users-of-java-agent-version-6-1-and-a-few-users-of-java-agent-version-6-2-0-to-6-4-2/141711) for additional details.
+</Callout>
+
+<Callout variant="important">
+  
+  Java agent versions greater than 6.4.2 no longer ship with SSL certificates bundled into the agent. You must provide your own SSL truststore to the agent by explicitly using [ca_bundle_path](/docs/agents/java-agent/configuration/java-agent-configuration-config-file#ca_config_bundle) configuration option or by using the default truststore provided by the JDK/JRE that the agent is running in (the agent will look to use the latter by default).
+  
+  Users wishing to add the global certificate to their local trust store can download the [DigiCert Global Root CA](https://www.digicert.com/kb/digicert-root-certificates.htm) from DigiCert.
+  
+  As the Java agent no longer provides its own SSL certififcates the `use_private_ssl` configuration option that enabled the bundled certificates has also been removed.
```

Comment:
This reads as though the cert has already been removed from versions 6.2.0+ (because of the section in the doc this appears under), I wonder if this would be confusing? Like the paragraph starting on line 26, should we start with the phrase "Java agents greater than 6.4.2 no longer provides..."

Reply:
Good point. I updated it to be phrased in a clearer manner.